packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -121,7 +121,7 @@ export async function findTeamMembersMatchingAttributeLogicOfRoute({
       getFieldResponse,
     });
 
-    if (!attributesQueryValue) {
+    if (raqbQueryValueUtils.isQueryValueEmpty(attributesQueryValue)) {
```

Comment:
attributesQueryValue could be an object but still without any rule. Better check for perf improvement.